### ISSUE TITLE
Can't load micro.owl with rdftab

### ISSUE THREAD

**Issue by turbomam** [ontodev/rdftab.rs#21]
I _assume_ the problem is in the `micro.owl` content, not in the `rdftab` logic. I can `robot convert` micro.owl to Turtle, but I get a `Content is not allowed in prolog` error message when trying to convert the Turtle back to RDF/XML

With `export RUST_BACKTRACE=full`:

`./rdftab multi_obo.db < micro.owl`

>thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: RdfXmlError { kind: Xml(EscapeError(UnrecognizedSymbol(1..4, Ok("obo")))) }', src/main.rs:57:5
stack backtrace:
   0:        0x10fa1de45 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h30b85a1761190f28
   1:        0x10fa3c44e - core::fmt::write::h5b0722e6ee659e34
   2:        0x10fa1c269 - std::io::Write::write_fmt::hf468289e762fa2f9
   3:        0x10fa20a9a - std::panicking::default_hook::{{closure}}::h836d46ca6b872224
   4:        0x10fa207bf - std::panicking::default_hook::h2afcf1998cd93f8c
   5:        0x10fa210ed - std::panicking::rust_panic_with_hook::he4f5d8b43533efd5
   6:        0x10fa20c82 - rust_begin_unwind
   7:        0x10fa3eacf - core::panicking::panic_fmt::h3559129da805eab4
   8:        0x10fa3eb45 - core::result::unwrap_failed::h170de03e7ee26a1a
   9:        0x10fa083a5 - rdftab::main::h1bc34813cbf130e1
  10:        0x10f9f76a6 - std::rt::lang_start::{{closure}}::h63a82885a43041b4
  11:        0x10fa20b58 - std::panicking::try::do_call::h29bd6a8b4eb65398
  12:        0x10fa2918b - __rust_maybe_catch_panic
  13:        0x10fa24389 - std::rt::lang_start_internal::h1cbb853ed77189ce
  14:        0x10fa08559 - main

**Comment by jamesaoverton** [comment]
I think this is the same problem as https://github.com/ontodev/rdftab.rs/issues/17. It's been fixed upstream and we just need to roll that out.

**Comment by jamesaoverton** [comment]
Hmm. Rolling out that change is a bit tricky, and will have to wait until I'm back from vacation.

As a workaround, can you run `robot convert -i micro.owl -o micro2.owl` and then see if RDFTab works on `micro2.owl`?

**Comment by turbomam** [comment]
sure will do

**Comment by cmungall** [comment]
Yes, there is a problem with the OWL file from micro, reported in 2019: https://github.com/carrineblank/MicrO/issues/22

Same problem with http://www.biopax.org/release/biopax-level3.owl -- but in this case I'm not sure there is anthing wrong with the OWL? In any case the problem is easily fixed with the roundtrip via robot
